pkgs/tools/wayland/wl-gammactl/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, stdenv, fetchFromGitHub
+, meson, pkg-config, ninja, git
+, wayland, wayland-scanner, wlroots, wlr-protocols, gtk3
+}:
+
+stdenv.mkDerivation rec {
+  pname = "wl-gammactl";
+  version = "unstable-2021-09-13";
+
+  src = fetchFromGitHub {
+    owner = "mischw";
+    repo = pname;
+    rev = "e2385950d97a3baf1b6e2f064dd419ccec179586";
+    sha256 = "8iMJK4O/sNIGPOBZQEfK47K6OjT6sxYFe19O2r/VSr8=";
+  };
+
+  nativeBuildInputs = [ meson pkg-config ninja git ];
```

Comment:
Why do we need git? It most likely needs special handling.

Reply:
There's a build step in the meson script that fetches the git submodule automatically but I patched it out.